src/v2i-hub/TelematicBridgePlugin/src/TelematicBridgeMsgWorker.h

```diff
@@ -238,7 +240,10 @@ namespace TelematicBridge
             }
             else
             {
-                json["payload"] = StringToJson(cJSON_Print(msg->payload));
+                // Render a cJSON entity to text for transfer/storage. Free the char* when finished.
+                char * payloadPtr = cJSON_Print(msg->payload);
```

Comment:
Can we replace this with a smart pointer to remove the need for free?

Reply:
It only returns a char* Type. How do we put this in smart pointer?